roachtest: django failed

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2254917&tab=buildLog) on [release-20.1@273b49d542d897d6f3b4f06d52322cbe77b0b436](https://github.com/cockroachdb/cockroach/commits/273b49d542d897d6f3b4f06d52322cbe77b0b436):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:202,django.go:214,test_runner.go:754: 
		Tests run on Cockroach v20.1.5-23-g273b49d
		Tests run against django cockroach-3.1.x
		8438 Total Tests Run
		8437 tests passed
		1 test failed
		554 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_json_field (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2254917&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2258496&tab=buildLog) on [release-20.1@e0256086e4f37cf120a86f8684072e91fee5b692](https://github.com/cockroachdb/cockroach/commits/e0256086e4f37cf120a86f8684072e91fee5b692):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:202,django.go:214,test_runner.go:754: 
		Tests run on Cockroach v20.1.5-36-ge025608
		Tests run against django cockroach-3.1.x
		8438 Total Tests Run
		8437 tests passed
		1 test failed
		554 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_json_field (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2258496&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2262010&tab=buildLog) on [release-20.1@765b36874f0ff4b898499eeda1dc5132cf491385](https://github.com/cockroachdb/cockroach/commits/765b36874f0ff4b898499eeda1dc5132cf491385):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:202,django.go:214,test_runner.go:754: 
		Tests run on Cockroach v20.1.5-43-g765b368
		Tests run against django cockroach-3.1.x
		8438 Total Tests Run
		8437 tests passed
		1 test failed
		554 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_json_field (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2262010&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2264895&tab=buildLog) on [release-20.1@63cd4307bf21758d0d6c5663b39b8a73020843fd](https://github.com/cockroachdb/cockroach/commits/63cd4307bf21758d0d6c5663b39b8a73020843fd):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:202,django.go:214,test_runner.go:754: 
		Tests run on Cockroach v20.1.5-45-g63cd430
		Tests run against django cockroach-3.1.x
		8438 Total Tests Run
		8437 tests passed
		1 test failed
		554 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_json_field (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2264895&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2267783&tab=buildLog) on [release-20.1@310ba51c4c75d351c5fdb7ec98f57be21b45b751](https://github.com/cockroachdb/cockroach/commits/310ba51c4c75d351c5fdb7ec98f57be21b45b751):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:202,django.go:214,test_runner.go:754: 
		Tests run on Cockroach v20.1.5-47-g310ba51
		Tests run against django cockroach-3.1.x
		8438 Total Tests Run
		8437 tests passed
		1 test failed
		554 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_json_field (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2267783&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2270945&tab=buildLog) on [release-20.1@295173db9014d59057b7733024848a806982613a](https://github.com/cockroachdb/cockroach/commits/295173db9014d59057b7733024848a806982613a):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:202,django.go:214,test_runner.go:754: 
		Tests run on Cockroach v20.1.5-51-g295173d
		Tests run against django cockroach-3.1.x
		8438 Total Tests Run
		8437 tests passed
		1 test failed
		554 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_json_field (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2270945&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2273968&tab=buildLog) on [release-20.1@9910e7188c024120ecf8d0ffe64c8e58c2a4b3c0](https://github.com/cockroachdb/cockroach/commits/9910e7188c024120ecf8d0ffe64c8e58c2a4b3c0):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:202,django.go:214,test_runner.go:754: 
		Tests run on Cockroach v20.1.5-57-g9910e71
		Tests run against django cockroach-3.1.x
		8441 Total Tests Run
		8440 tests passed
		1 test failed
		554 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_json_field (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2273968&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2277577&tab=buildLog) on [release-20.1@be8c0a720e98a147263424cc13fc9bfc75f46013](https://github.com/cockroachdb/cockroach/commits/be8c0a720e98a147263424cc13fc9bfc75f46013):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:202,django.go:214,test_runner.go:754: 
		Tests run on Cockroach v20.1.5-59-gbe8c0a7
		Tests run against django cockroach-3.1.x
		8441 Total Tests Run
		8440 tests passed
		1 test failed
		554 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_json_field (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2277577&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2280937&tab=buildLog) on [release-20.1@56a0bd4a62fa28d52521e51996958965b3c914d2](https://github.com/cockroachdb/cockroach/commits/56a0bd4a62fa28d52521e51996958965b3c914d2):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:202,django.go:214,test_runner.go:754: 
		Tests run on Cockroach v20.1.5-61-g56a0bd4
		Tests run against django cockroach-3.1.x
		8441 Total Tests Run
		8440 tests passed
		1 test failed
		554 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_json_field (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2280937&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2283723&tab=buildLog) on [release-20.1@6f671e63f52f277b79faae939a11bb746b413c40](https://github.com/cockroachdb/cockroach/commits/6f671e63f52f277b79faae939a11bb746b413c40):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:202,django.go:214,test_runner.go:754: 
		Tests run on Cockroach v20.1.5-65-g6f671e6
		Tests run against django cockroach-3.1.x
		8441 Total Tests Run
		8440 tests passed
		1 test failed
		554 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_json_field (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2283723&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2286860&tab=buildLog) on [release-20.1@85c8a57e88141fc99ce00aa4a3e6cc5079a1246a](https://github.com/cockroachdb/cockroach/commits/85c8a57e88141fc99ce00aa4a3e6cc5079a1246a):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:202,django.go:214,test_runner.go:755: 
		Tests run on Cockroach v20.1.5-67-g85c8a57
		Tests run against django cockroach-3.1.x
		8441 Total Tests Run
		8440 tests passed
		1 test failed
		554 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_json_field (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2286860&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2287963&tab=buildLog) on [release-20.1@85c8a57e88141fc99ce00aa4a3e6cc5079a1246a](https://github.com/cockroachdb/cockroach/commits/85c8a57e88141fc99ce00aa4a3e6cc5079a1246a):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:202,django.go:214,test_runner.go:755: 
		Tests run on Cockroach v20.1.5-67-g85c8a57
		Tests run against django cockroach-3.1.x
		8442 Total Tests Run
		8441 tests passed
		1 test failed
		553 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_json_field (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2287963&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2316604&tab=buildLog) on [release-20.1@67699d3b5e40ad38e3b6ff46248a5156e322adb2](https://github.com/cockroachdb/cockroach/commits/67699d3b5e40ad38e3b6ff46248a5156e322adb2):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:202,django.go:214,test_runner.go:755: 
		Tests run on Cockroach v20.1.5-69-g67699d3
		Tests run against django cockroach-3.1.x
		8441 Total Tests Run
		8440 tests passed
		1 test failed
		554 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_json_field (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2316604&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2322256&tab=buildLog) on [release-20.1@1ee84232a825c9fd69437c00c1dbe2dfb950d009](https://github.com/cockroachdb/cockroach/commits/1ee84232a825c9fd69437c00c1dbe2dfb950d009):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:202,django.go:214,test_runner.go:755: 
		Tests run on Cockroach v20.1.6-14-g1ee8423
		Tests run against django cockroach-3.1.x
		8441 Total Tests Run
		8440 tests passed
		1 test failed
		554 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_json_field (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2322256&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2324598&tab=buildLog) on [release-20.1@a84063ed9e7e3c43c710534629399bb957f17cca](https://github.com/cockroachdb/cockroach/commits/a84063ed9e7e3c43c710534629399bb957f17cca):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:202,django.go:214,test_runner.go:755: 
		Tests run on Cockroach v20.1.6-19-ga84063e
		Tests run against django cockroach-3.1.x
		8441 Total Tests Run
		8440 tests passed
		1 test failed
		554 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_json_field (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2324598&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2328549&tab=buildLog) on [release-20.1@5a4e41010f8faa095c88e2e5b142e0b6880c9f44](https://github.com/cockroachdb/cockroach/commits/5a4e41010f8faa095c88e2e5b142e0b6880c9f44):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:202,django.go:214,test_runner.go:755: 
		Tests run on Cockroach v20.1.6-21-g5a4e410
		Tests run against django cockroach-3.1.x
		8442 Total Tests Run
		8441 tests passed
		1 test failed
		554 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_json_field (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2328549&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2332347&tab=buildLog) on [release-20.1@d89343cdb6dcd86584584b098889795620dfd534](https://github.com/cockroachdb/cockroach/commits/d89343cdb6dcd86584584b098889795620dfd534):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:202,django.go:214,test_runner.go:755: 
		Tests run on Cockroach v20.1.6-29-gd89343c
		Tests run against django cockroach-3.1.x
		8442 Total Tests Run
		8441 tests passed
		1 test failed
		554 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_json_field (unexpected)
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2332347&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2336201&tab=buildLog) on [release-20.1@05fa4f5ac66d1672cc9f33752eea497ea163193d](https://github.com/cockroachdb/cockroach/commits/05fa4f5ac66d1672cc9f33752eea497ea163193d):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:202,django.go:214,test_runner.go:755: 
		Tests run on Cockroach v20.1.6-33-g05fa4f5
		Tests run against django cockroach-3.1.x
		8442 Total Tests Run
		8441 tests passed
		1 test failed
		554 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_json_field (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2336201&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2338823&tab=buildLog) on [release-20.1@0a1f9df246ed0c6f331561fff653398449cb2c41](https://github.com/cockroachdb/cockroach/commits/0a1f9df246ed0c6f331561fff653398449cb2c41):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:202,django.go:214,test_runner.go:755: 
		Tests run on Cockroach v20.1.6-38-g0a1f9df
		Tests run against django cockroach-3.1.x
		8442 Total Tests Run
		8441 tests passed
		1 test failed
		554 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_json_field (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2338823&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

[(roachtest).django failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=2341061&tab=buildLog) on [release-20.1@406792b58d32dc053008cc129b409252798f509c](https://github.com/cockroachdb/cockroach/commits/406792b58d32dc053008cc129b409252798f509c):

```
The test failed on branch=release-20.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/django/run_1
	orm_helpers.go:228,orm_helpers.go:154,django.go:202,django.go:214,test_runner.go:755: 
		Tests run on Cockroach v20.1.6-42-g406792b
		Tests run against django cockroach-3.1.x
		8442 Total Tests Run
		8441 tests passed
		1 test failed
		554 tests skipped
		21 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- SKIP: backends.mysql.test_creation.DatabaseCreationTests.test_create_test_db_unexpected_error due to MySQL tests (expected)
		--- FAIL: inspectdb.tests.InspectDBTestCase.test_json_field (unexpected)
		For a full summary look at the django artifacts 
		
		An updated blocklist (djangoBlocklist20_1) is available in the artifacts' django log

```

<details><summary>More</summary><p>


Artifacts: [/django](https://teamcity.cockroachdb.com/viewLog.html?buildId=2341061&tab=artifacts#/django)

[See this test on roachdash](https://roachdash.crdb.dev/?filter=status%3Aopen+t%3A.%2Adjango.%2A&sort=title&restgroup=false&display=lastcommented+project)
<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub></p></details>

--- a/pkg/cmd/roachtest/django_blocklist.go
+++ b/pkg/cmd/roachtest/django_blocklist.go
@@ -169,7 +169,9 @@ var djangoBlocklists = blocklistsForVersion{
 // Maintain that this list is alphabetized.
 var djangoBlocklist20_2 = blocklist{}
 
-var djangoBlocklist20_1 = blocklist{}
+var djangoBlocklist20_1 = blocklist{
+	"inspectdb.tests.InspectDBTestCase.test_json_field": "unknown",
+}
 
 var djangoIgnoreList20_2 = djangoIgnoreList20_1
 

--- a/pkg/cmd/roachtest/sqlalchemy.go
+++ b/pkg/cmd/roachtest/sqlalchemy.go
@@ -83,7 +83,7 @@ func runSQLAlchemy(ctx context.Context, t *test, c *cluster) {
 	}
 
 	if err := repeatRunE(ctx, c, node, "install pytest", `
-		sudo pip3 install --upgrade --force-reinstall setuptools pytest pytest-xdist psycopg2
+		sudo pip3 install --upgrade --force-reinstall setuptools pytest==6.0.1 pytest-xdist psycopg2
 	`); err != nil {
 		t.Fatal(err)
 	}

